Update README.md to pin istio version (#3)
rvennam/ambient-multicluster-workshop: README.md

```

> Cluster1 will act as the management cluster and workload cluster: (note: mgmt-values.yaml can be found in this repo)

Reviewer: ```suggestion
Cluster1 will act as the management cluster and workload cluster: (see [mgmt-values.yaml](./mgmt-values.yaml) for reference)
```
